astropy/utils/tests/test_parsing.py

```diff
@@ -35,6 +36,9 @@ def t_NUMBER(t):
                 t.value = int(t.value)
                 return t
 
+            def t_error(t):
```

Comment:
Is this being required an indication of anything more serious?

Reply:
I don't think so. Just that PLY lacks a sensible default for `t_error` if the lexer does not provide one.